Automattic/wp-calypso
# Fix Stepper performance bug

In another instance of tree-shaking breaking down, Stepper was pulling a ton of JS and CSS that it never runs since https://github.com/Automattic/wp-calypso/pull/67887.

Fixes https://github.com/Automattic/wp-calypso/issues/97337

## Proposed Changes

This moves the `setupErrorLogger` function from a big file full of imports into its own file to help tree shaking.

## Why are these changes being made?
To make Stepper faster. 

## Testing Instructions

1. Go to /setup/onboarding.
2. Make sure dashicons.css is not loaded. 

### Regression testing
~~Follow the testing steps of https://github.com/Automattic/wp-calypso/pull/67887~~
These steps don't pass on prod.
